src/bundle/Resources/views/admin/content_type/create.html.twig

```diff
@@ -70,12 +70,20 @@
                 {% for field_definition in form.fieldDefinitionsData %}
                     {% set value = field_definition.vars.value %}
 
-                    <div class="card ez-card ez-card--fieldtype-container">
+                    <div class="card ez-card ez-card--fieldtype-container mb-3 ez-card--collapsed">
                         <div id="field-definition-{{ value.identifier }}" class="ez-card__header ez-card__header--secondary">
                             {{ form_widget(field_definition.selected, {
                                 'label': '%s (%s)' | format(value.names[language_code], value.fieldTypeIdentifier),
                                 'label_attr': {class: 'checkbox-inline'}
                             }) }}
+                            <button class="ez-card__body-display-toggler btn">
+                                <svg class="ez-icon ez-icon--medium ez-icon--caret-down" aria-hidden="true">
```

Comment:
I think the `aria-hidden` is not correctly set. Both are hidden? If you want to use `aria-hidden` you should also change the state of it when user clicks on the button and change the state.

Reply:
Well, I disagree. By using any `aria-*` attributes you're informing screen readers about the element purpose. In this case, when using `aria-hidden` it doesn't mean it's visually hidden. It means that screen readers can omit this piece of HTML code and process the next HTML nodes.

Reply:
According to the W3C recommendations: 
```
aria-hidden (state)§

Indicates whether the element is exposed to an accessibility API. See related aria-disabled.

User agents determine an element's hidden status based on whether it is rendered, and the rendering is usually controlled by CSS. For example, an element whose display property is set to none is not rendered. An element is considered hidden if it, or any of its ancestors are not rendered or have their aria-hidden attribute value set to true.
```
ref: https://www.w3.org/TR/wai-aria-1.1/#aria-hidden

So as I understand this if you set `aria-hidden=true` the screen reader will omit this element.

Reply:
Yes, it hidden for screen readers only.

Reply:
So on a screen reader, you will not be able to expand the card?

Reply:
It's a screen reader, it won't try to read the code of the icon. The button will still work. If we try to increase the accessibility of our interface maybe we should start applying correct `aria` attributes here and there in our codebase.
In this case, it just hides an icon from reading it.